app/src/main/kotlin/cz/covid19cz/erouska/App.kt

```diff
@@ -12,6 +13,7 @@ class App : BaseApp(), KoinComponent {
     override fun onCreate() {
         super.onCreate()
         setupKoin()
+        FirebaseApp.initializeApp(this)
```

Comment:
Proč je tohle potřeba? Imho se Firebase inicializuje sama.

Reply:
řešil jsem nějaké problémy s Firebase, když jsme neměli whitelisted appId, už je to zbytečné